context-propagation/pom.xml

```diff
@@ -35,7 +35,7 @@
         <dependency>
             <groupId>io.smallrye</groupId>
             <artifactId>smallrye-context-propagation</artifactId>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
Should we rename the module to `mutiny-smallrye-context-propagation`?

Reply:
That's what we need to discuss and maybe have both.

Reply:
After reflection, I'm just going to rename the module to mutiny-smallrye-context-propagation. If someone needs a pure implementation on top of the spec, we can restore the old code.

Reply:
Agreed